### DISCUSSION TITLE
iOS PWA with PWABuilder

### DISCUSSION THREAD

**Discussion by fenbru** [add2cal/add-to-calendar-button#462] [Q&A]
Hi guys,

Has anyone tried `atcb` in an iOS PWA app built with [PWABuilder](https://www.pwabuilder.com/)?

Many thanks!

**Comment by fenbru** [comment]
> Hi guys,
> 
> Has anyone tried `atcb` in an iOS PWA app built with [PWABuilder](https://www.pwabuilder.com/)?
> 
> Many thanks!

Any response about this?

**Comment by jekuer** [comment]
The button works fine with iOS PWA apps.
However, I am not sure about this PWABuilder tool. Do you have a specific problem with it?

**Reply by fenbru** [reply]
It's a native wrapper around the PWA app, so when you click on the `atcb` the outcome is not the same as on a regular PWA app. Just wanted to know if someone had any exp around this.
